<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>                             SAYI TAHMİN OYUNU</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>31</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>       1.Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>              2.Aralık</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aralik1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>51</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="aralik2">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>51</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_uret">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Edilecek Sayı Üret</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tahmin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Et</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_sonuc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
